SDK/MobileCenter/Microsoft.Azure.Mobile.Windows.Shared/Channel/Channel.cs

```diff
@@ -43,12 +43,11 @@ public sealed class Channel : IChannelUnit
             _batchScheduled = false;
             _enabled = true;
             DeviceInformationHelper.InformationInvalidated += (sender, e) => InvalidateDeviceCache();
-            Task.Run(CountFromDiskAsync);
```

Comment:
I'm confused because we enter channel in an enabled state and start using it without ever calling `ResumeAsync`. Should this be invoked when it is created? And if so, how will you ensure that it is called first? Perhaps create the mutex in a locked state, and then unlock it at the end of an initialization function (that doesn't lock it but assumes mutex is already locked)